docs/builders/exoscale.md

```diff
@@ -62,6 +62,9 @@ other OS, we recommend using the [QEMU][packerqemu] plugin combined with the
   will be created before creating the instance, and destroyed after a
   successful build.
 
+- `snapshot_download` (boolean) - Whether to download a snapshot of the compute
+  instance. Defaults to `false`.
+
```

Comment:
```suggestion
- `snapshot_download` (boolean) - Whether to download a snapshot of the compute
  instance. Defaults to `false`.
```
It would be nice allowing the user to specify a `snapshot_download_path` too (defaulting to `./output`)

Reply:
Hi @exo-cedric , thank you for the suggestion. Added an option to specify path where to download the snapshot -> [`4a2b99c` (#27)](https://github.com/exoscale/packer-plugin-exoscale/pull/27/commits/4a2b99c29c860e4aceed4475c6e4bb37f4ba2bb8)